### DISCUSSION TITLE
Other Languages

### DISCUSSION THREAD

**Discussion by fkbranco** [drakkan/sftpgo#1016] [Ideas]
Drakkan, I wonder if it's possible to translate the Client UI.

I'd feel very happy to help on this matter (at least in portuguese, my mother language)..

Best

**Comment by drakkan** [comment]
Hi,

this was already discussed many times

https://github.com/drakkan/sftpgo/issues/998#issuecomment-1257915010
https://github.com/drakkan/sftpgo/issues/949

in short I need a sponsorship to add what I proposed in the first linked comment, thanks
